render-thumb-for.sh

```diff
@@ -98,6 +99,35 @@ file_type_get () {
     fi
 }
 
+paint () {
+    img_path=$1
+    img_wh=$2
+
+    if [[ -n "$is_cmd_kitten" ]]; then
```

Comment:
`kitten` is available system-wide, so this tries to use `kitten` even when not using Kitty thus breaking sixel.

Reply:
https://github.com/iambumblehead/render-thumb-for.sh/pull/24